src/Compiler/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X/pack/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X.Internal.Transport.csproj

```diff
@@ -1,16 +0,0 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <Description>Transport package for Razor extension binaries. For internal use only.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <IsShipping>false</IsShipping>
-    <IncludeBuildOutput>false</IncludeBuildOutput>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <ProjectReferemce Include="..\src\Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X.csproj" ReferenceOutputAssembly="false" />
-
-    <Content Include="$(ArtifactsDir)bin\Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X\$(Configuration)\netstandard2.0\Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X.dll" PackagePath="lib\$(TargetFramework)" />
```

Comment:
❔ Is this just a DLL from a different project in the repo? Still used, but not as part of this project.

Reply:
That's correct. Not sure what your concern is here?

Reply:
'❔' designates a question, not a concern. Sometimes answering the question can reveal the need to make a change, but many times answering is all that's requested.